main/src/cgeo/geocaching/location/GeopointFormatter.java

```diff
@@ -98,8 +98,8 @@ public static String format(final Format format, final Geopoint gp) {
 
             case LAT_LON_DECMINUTE_SHORT:
```

Comment:
The example in the enum comment also needs to be adapted and should mention that the string in unlocalized.